dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -125,7 +125,38 @@ This is the creative that your Ad Ops team needs to upload to the ad server (it'
 {: .alert.alert-success :}
 You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html).
 
-{% include dev-docs/amp-creative.md %}
+{: .alert.alert-danger :}
+Replace **MACRO** in the code below with the appropriate macro for the ad server you're using. For example, for DFP replace **MACRO** with **PATTERN**.
```

Comment:
Many people are going to find this confusing. They generally copy-paste from the site. I think we need a specific version for GAM/DFP with a comment that says to change it for other ad servers.

Reply:
Agreed.  Let's make this code block DFP/ GAM specific and maybe include a note that indicates that the key-value macro may be different for other ad servers.

Reply:
I tried to make this more general to encompass all ad servers. I'll break it out. But if we're going to change it here then we should probably also have separate examples to point to here: https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html